apps/openmw/mwworld/store.cpp

```diff
@@ -1053,6 +1053,34 @@ namespace MWWorld
         }
     }
 
+    template<>
+    void Store<ESM::Static>::setUp()
+    {
+        // Load default marker definitions, if game files do not have them for some reason
+        std::pair<std::string, std::string> markers[] = {
+            std::make_pair("divinemarker", "marker_divine.nif"),
+            std::make_pair("doormarker", "marker_arrow.nif"),
+            std::make_pair("northmarker", "marker_north.nif"),
+            std::make_pair("templemarker", "marker_temple.nif"),
+            std::make_pair("travelmarker", "marker_travel.nif")
+        };
+
+        for (const std::pair<std::string, std::string> marker : markers)
+        {
+            if (search(marker.first) == 0)
+            {
+                ESM::Static newMarker = ESM::Static(marker.first, marker.second);
+                mStatic.insert(std::make_pair(marker.first, newMarker));
```

Comment:
For future reference, insert returns an interator (actually a pair with first being the iterator) to the new element. Therefore the find below is unnecessary.